e2e-tests/tests/420-text-plugin.ts

```diff
@@ -4,32 +4,52 @@ Feature('Serlo Editor - Text plugin basic interactions')
 
 Before(popupWarningFix)
 
-// First Text plugin is the multimedia explanation,
-// second is the default empty Text plugin.
 const initialTextPluginCount = 2
+const pageUrl = '/entity/create/Article/1377'
+const selectors = {
+  textEditor: '$plugin-text-editor',
+  addNewPluginButton: '$add-new-plugin-row-button',
+  multimediaWrapper: '$plugin-multimedia-wrapper',
+  toolbarUndoButton: '$editor-toolbar-undo',
+  toolbarRedoButton: '$editor-toolbar-redo',
+  numberedListButton: '$plugin-toolbar-button-nummerierte-liste',
+}
+
+function addNewTextPlugin(I) {
```

Comment:
Nice idea with the helper functions. Maybe even consider creating a file for editor helpers and exporting all of them from there?

Reply:
Created a ticket for this: https://linear.app/serlo/issue/PE-61/create-helper-functions-for-tests